src/IIIFPresentation/Models/API/Collection/FlatCollection.cs

```diff
@@ -1,48 +1,45 @@
-﻿using IIIF.Presentation.V3.Strings;
+﻿using IIIF.Presentation.V3;
+using IIIF.Presentation.V3.Strings;
 using Newtonsoft.Json;
 
 namespace Models.API.Collection;
 
-public class FlatCollection
+public class FlatCollection : IIIF.Presentation.V3.Collection
 {
     [JsonProperty("@context")]
-    public List<string>? Context { get; set; }
-    
-    public string? Id { get; set; }
-    
-    public string? PublicId { get; set; }
-    
-    public PresentationType Type { get; set; }
+    public new List<string>? Context
+    {
+        get => base.Context as List<string>;
+        set => base.Context = value;
+    }
+
+    #region Overrides of Collection
 
-    public List<string> Behavior { get; set; } = new ();
+    public override string Type { get; } = nameof(FlatCollection);
```

Comment:
I think this should just be `collection`

Reply:
then there is no way for the `ResourceBaseV3Converter` to know which concrete class deriving from collection to use

Reply:
might need to talk to Tom about it then, as I think the [docs](https://deploy-preview-3--dlcs-docs.netlify.app/api-doc/iiif#public-and-api-storage-roots) says it should be `Collection`